<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget{
	background-color: #C0D8D7;
}</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="HomeScreen">
    <widget class="QLabel" name="SudokuLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sudoku.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statsButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Leaderboards</string>
     </property>
    </widget>
    <widget class="QPushButton" name="howtoButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>How to play ?</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="NewGameScreen">
    <widget class="QLabel" name="newgameLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="easyButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Easy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="midButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hardButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hard</string>
     </property>
    </widget>
    <widget class="QPushButton" name="continueButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QLabel" name="loadsavedLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load Saved</string>
     </property>
    </widget>
    <widget class="QPushButton" name="returnBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="PlayScreen">
    <widget class="QLCDNumber" name="playDuration">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="doneButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>430</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QPushButton" name="solveButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>480</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Solve</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save and Exit</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="gameGrid">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>392</width>
       <height>366</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>43</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Line{
	color: #446688;
}</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Line{
	color: #446688;
}</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>20</width>
       <height>364</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Line{
	color: #446688;
}</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>20</width>
       <height>364</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Line{
	color: #446688;
}</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="StatsScreen">
    <widget class="QLabel" name="statsLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Leaderboards</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>371</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="statsDisplay"/>
    </widget>
    <widget class="QPushButton" name="returnBtn_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="endScreen">
    <widget class="QLabel" name="completeLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Win Label</string>
     </property>
    </widget>
    <widget class="QLabel" name="completeTime">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time: 00:00 </string>
     </property>
    </widget>
    <widget class="QLabel" name="completeDifficulty">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Difficulty: Easy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="completeHomeButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="completeStatsButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>380</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Leaderboards</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HowToScreen">
    <widget class="QLabel" name="howToLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>How to play Sudoku</string>
     </property>
    </widget>
    <widget class="QLabel" name="instructions">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>411</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	padding: 10px;
}</string>
     </property>
     <property name="text">
      <string>The rules for Sudoku are simple. 
 A 9×9 square must be filled in with 
 numbers from 1-9 with no repeated numbers in each line, 
 horizontally (columns) or vertically (rows). 
 To challenge you more, there are 3×3 squares 
 marked out in the grid (blocks), and each of 
 these squares can't have any repeat numbers either.

When the puzzle starts a few of the cells will be 
 already filled in. 
 The easy puzzles will have more numbers filled in 
 while the harder puzzles have less numbers filled in. 
 The goal of the game is to fill in every cell in the 
 grid by following the above rule of not repeating a 
 number in a block, a row or a column. 

 Whoever fills every cell in the grid without a mistake 
 in the least amount of time wins the game. 
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="returnBtn_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
